### PR TITLE
[TEST] Added unit tests, increase code coverage on package org.pentah…

### PR DESCRIPTION
**Pull request by niehusa**
…o.platform.api.engine

### CODE REVIEW

File: api/test-src/org/pentaho/platform/api/engine/ActionValidationExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.Constructor;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.pentaho.actionsequence.dom.IActionDefinition;
+
+public class ActionValidationExceptionTest {
+//  @Test
```
Comment (lucboudreau) [inline api/test-src/org/pentaho/platform/api/engine/ActionValidationExceptionTest.java]: Please delete commented out code.

**Comment by niehusa** [comment]
@pedrofvteixeira Please review this report and merge the PR if looks okay to you.

**Comment by pedrofvteixeira** [comment]
PR looks good, unit tests pass and coverage is up 👍 

Please address Luc's comment and sanitize that class; rebase, and I'll process it afterwards. Thanks!

**Comment by pedrofvteixeira** [comment]
PR looks good to me. Merging

**Pull request merged**